app/Nova/Metrics/TotalCollections.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace App\Nova\Metrics;
+
+use App\Payment;
+use Illuminate\Http\Request;
+use Laravel\Nova\Metrics\Value;
+
+class TotalCollections extends Value
```

Comment:
After thinking about this metric more, I think it should be split into two: one showing total transaction _count_ and one showing a breakdown of payment methods.

There's a more detailed report in my head but it doesn't fit in a nova metric.

Reply:
Correction: just make it a payment method breakdown like RSVP source.

Reply:
The breakdown will have the total count on it as well as the counts for each individual type.

Reply:
Added in e4051cb.